src/utils/render.tsx

```diff
@@ -197,21 +207,20 @@ export const formatValueWithLink = (props: { value: number; onClick: () => void
 export const formatQueryForNaturalLanguage = (platform: string, network: string, entity: string, query: ConseilQuery, attributes: AttributeDefinition[], operators: any) => {
     const timestamp = (query.predicates && query.predicates.length && query.predicates.find((p: any) => p.field === 'timestamp')) || null;
     const filters = (query.predicates && query.predicates.length && query.predicates.filter((f: any) => f.field !== 'timestamp')) || [];
-    let title = entity.slice(0, 1).toLocaleUpperCase() + entity.slice(1);
     let renderTimestamp;
 
     if (timestamp) {
         const attribute = attributes.filter(a => a.name === timestamp.field)[0] as AttributeDefinition;
-
-        const operation = operators[getOperatorType(attribute.dataType)].filter((o: any) => o['name'] === timestamp.operation)[0]['displayName'];
-        const value = formatValueForDisplay(platform, network, timestamp.field, timestamp.set[0], attribute, () => {}, undefined, true);
+        const operation = operators[getOperatorType(attribute.dataType)].filter((o: any) => !timestamp.inverse ? o['name'] === timestamp.operation : o['name'] === 'not' + timestamp.operation)[0]['displayName'];
+        const shouldShowTime = moment.default(Number(timestamp.set[0])).hour() > 0 || moment.default(Number(timestamp.set[0])).minute() > 0;
+        const value = formatValueForDisplay(platform, network, timestamp.field, timestamp.set[0], attribute, () => {}, undefined, true, false, `${shouldShowTime ? 'HH:mm a on' : ''} MMMM Do, YYYY`);
```

Comment:
We should do this differently, date format comes from metadata, let's not override it.